server/survey/experiments/previous.experiment.js

```diff
@@ -0,0 +1,1220 @@
+      reasoning: "Knowledge",
+      category: "General reference",
+    },
+    params: {
+      ids: [
+        3487, 2506, 3866, 3101, 139, 2444, 2922, 683, 279, 3108, 274, 42, 3746,
+        1165, 3720,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Social",
+      everyday: "Everyday",
+      figure_of_speech: "Literal language",
+      judgment: "Positive",
+      opinion: "Fact",
+      reasoning: "Reasoning",
+      category: "People and self",
+    },
+    params: {
+      ids: [
+        286, 2254, 1975, 2022, 79, 1594, 1984, 1410, 1008, 3394, 1721, 1857,
+        2201, 1895, 836,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Social",
+      everyday: "Everyday",
+      figure_of_speech: "Literal language",
+      judgment: "Positive",
+      opinion: "Fact",
+      reasoning: "Reasoning",
+      category: "Human activities",
+    },
+    params: {
+      ids: [
+        1651, 370, 1763, 2455, 3182, 1503, 1740, 1564, 1574, 1923, 3809, 1179,
+        2235, 1906, 3370,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Physical",
+      everyday: "Abstract",
+      figure_of_speech: "Literal language",
+      judgment: "Positive",
+      opinion: "Fact",
+      reasoning: "Reasoning",
+      category: "History and events",
+    },
+    params: {
+      ids: [
+        57, 4057, 1476, 237, 807, 27, 1366, 4191, 751, 3741, 517, 2839, 1341,
+        1111, 1188,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Social",
+      everyday: "Abstract",
+      figure_of_speech: "Literal language",
+      judgment: "Normative",
+      opinion: "Opinion",
+      reasoning: "Reasoning",
+      category: "Religion and belief systems",
+    },
+    params: {
+      ids: [
+        200, 2364, 2511, 3852, 601, 2658, 2703, 2400, 2443, 2533, 63, 4325, 632,
+        2459, 2704,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Physical",
+      everyday: "Everyday",
+      figure_of_speech: "Literal language",
+      judgment: "Positive",
+      opinion: "Fact",
+      reasoning: "Reasoning",
+      category: "General reference",
+    },
+    params: {
+      ids: [
+        146, 3252, 2906, 66, 2410, 2934, 1280, 1433, 2836, 3684, 746, 1201,
+        3794, 1100, 1153,
+      ],
+    },
+    function: GetStatementById,
+  },
+  {
+    design_point: {
+      behavior: "Social",
+      everyday: "Everyday",
+      figure_of_speech: "Figure of speech",
+      judgment: "Normative",
+      opinion: "Opinion",
+      reasoning: "Reasoning",
+      category: "Culture and the arts",
+    },
+    params: {
+      ids: [
+        3453, 1659, 2680, 1123, 1552, 2889, 1805, 2346, 541, 1534, 3772, 179,
+        254, 1848, 1727,
+      ],
+    },
+    function: GetStatementById,
+  },
+];
+
+const experiment = {
```

Comment:
Shouldn't there be some kind of assignment handling here? e.g., so turk people can see this experiment and others can not?

Reply:
Thanks for noticing. fixed now.